### PR TITLE
openstack: development checkpoint 

### PR DESCRIPTION
**Pull request by bennyz**
This PR is a checkpoint as a plan can currently be added and reconciled without crashing

### CODE REVIEW

File: pkg/controller/map/network/handler/openstack/handler.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+	"github.com/konveyor/forklift-controller/pkg/controller/watch/handler"
+	liberr "github.com/konveyor/forklift-controller/pkg/lib/error"
+	libweb "github.com/konveyor/forklift-controller/pkg/lib/inventory/web"
+	"github.com/konveyor/forklift-controller/pkg/lib/logging"
+	"golang.org/x/net/context"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+)
+
+// Package logger.
+var log = logging.WithName("networkMap|ovirt")
```
Comment (ahadas) [inline pkg/controller/map/network/handler/openstack/handler.go:18]:
```suggestion
var log = logging.WithName("networkMap|openstack")
```

File: pkg/controller/map/network/handler/openstack/handler.go [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+			r.Provider().Name),
+		"watch",
+		w.ID())
+
+	return
+}
+
+// Resource created.
+func (r *Handler) Created(e libweb.Event) {
+	if network, cast := e.Resource.(*ovirt.Network); cast {
```
Comment (ahadas) [inline pkg/controller/map/network/handler/openstack/handler.go:49]: do we really set it to ovirt.Network or how is it that it doesn't fail?

File: pkg/controller/provider/container/openstack/client.go [hunk 3]
```diff
@@ -46,8 +46,8 @@ const (
 
 // Client struct
 type Client struct {
-	url                 string
-	secret              *core.Secret
+	URL                 string
+	Secret              *core.Secret
```
Comment (ahadas) [inline pkg/controller/provider/container/openstack/client.go:50]: for visibility, right?

File: pkg/controller/map/network/handler/openstack/handler.go (hunk 2 above)
Comment (bennyz) [reply pkg/controller/map/network/handler/openstack/handler.go:49]: I suppose it isn't called yet 😅

File: pkg/controller/provider/container/openstack/client.go (hunk 3 above)
Comment (bennyz) [reply pkg/controller/provider/container/openstack/client.go:50]: yes, to make it usable in the plan

**Review by ahadas — COMMENTED** [review]
lgtm but need to fix that issue that is reported by bazel

**Review by ahadas — APPROVED** [review]

**Pull request merged**